src/Controllers/Dispatcher.ts

```diff
@@ -3,14 +3,25 @@ import { RecordController } from './RecordController';
 import { EventOrigin } from '../Interfaces/BoardInterfaces';
 import { EventType, IEvent } from '../Interfaces/AppInterfaces';
 import Timer from '../Timer/Timer';
-import { IAction, Targets, PointerActionType, IZoomOptions, IPointerEvent } from '../Interfaces/ActionInterfaces';
+import { IAction, Targets, PointerActionType, IZoomOptions, IPointerEvent, IStrokeProps } from '../Interfaces/ActionInterfaces';
 import { singleton } from 'tsyringe';
 import { IModifier, ModifierTarget } from '../Domain/Modifier';
 
 @singleton()
 export default class Dispatcher {
   private actionListeners: { (action: IAction): void }[] = [];
-  constructor(private state: DispatcherState, private timer: Timer, private recorder: RecordController) {}
+
+  private strokeProps: IStrokeProps = {
+    stroke: 'green',
+    'stroke-width': 50,
+    'buffer-size': 20,
+    fill: undefined,
+  };
+
+  constructor(private state: DispatcherState, private timer: Timer, private recorder: RecordController) {
+    console.log('initializing strokeProps ' + JSON.stringify(state.strokeProps));
+    this.strokeProps = state.strokeProps;
```

Comment:
Instead of settings strokeProps as a member variable, just set the state as a member variable and access it via this.state.strokeProps. Default stroke prop values should be set in the DispatcherState class.